Firestore/Swift/Tests/API/BasicCompileTests.swift

```diff
@@ -17,10 +17,11 @@
 // These aren't tests in the usual sense--they just verify that the Objective-C to Swift translation
 // results in the right names.
 
+@_exported import class FirebaseCore.Timestamp
+
```

Comment:
Does this file compile if you refer this import?

Reply:
Yes it did. However I checked the file content and found it makes more sense to `@_exported import class`. I will change that shortly, thank you!

Also the github CI fail in `spm-binary` build, do you have a clue what I missed?

Reply:
`spm-binary` is currently broken (unrelated to this PR!)

Reply:
Also, sorry. I edited my first message. I meant "remove" but I said "refer". SG!

Reply:
Also, I feel it makes sense to change the source file to use `@_exported` but it would be less value to do the same for test file. Am I understand correctly for the `@_exported`?

Reply:
> feel it makes sense to change the source file to use @_exported but it would be less value to do the same for test file.

Yes, agreed. If a test file compiles without the import, then removing it is fine. If a test needs the FirebaseCore import, doing `import FirebaseCore` instead of the `@_exported import ...` is fine since you're correct that `@_exported` doesn't make sense.

Reply:
Hi @ncooke3 , I have changed the PR, please let me know if there is anything can be improved :)

Reply:
Thanks, @cherylEnkidu. LGTM. One more small thing–– the part of the changelog entry that discusses the moved API can now be removed.

Reply:
Going to resolve this to trigger the merge. Changelog can always be fixed up later.